core-v1/src/test/scala/SimpleJustifiedExistsEvidenceSpec.scala

```diff
@@ -1,15 +1,30 @@
 package com.rallyhealth.vapors.v1
 
 import data.{Evidence, FactTable, Justified, NoEvidence, Window}
-import example.FactTypes
+import example.{CombinedTags, FactTypes}
 
 import cats.data.NonEmptyList
 import munit.FunSuite
 
+import scala.collection.immutable.SortedSet
+
 class SimpleJustifiedExistsEvidenceSpec extends FunSuite {
 
   import dsl.simple.justified._
 
+  test("Justified[Seq[_]].exists returns all evidence when empty".fail) {
+    val emptyTagsFact = FactTypes.CombinedTags(CombinedTags.now(SortedSet()))
+    val expr = valuesOfType(FactTypes.CombinedTags).exists {
+      _.getAs[Seq](_.select(_.tags)).exists {
+        _ === "ignore".const
+      }
+    }
+    val result = expr.run(FactTable(emptyTagsFact))
+    assert(!result.value)
+    // TODO: This test is marked as failed because the result should contain the original fact as evidence
+    assert(result.evidence.factSet.contains(emptyTagsFact))
+  }
```

Comment:
Not sure why I added this test, but I think this was a note for me to figure out why this test is failing. I think this should be done in a separate PR though. This one is big enough already and this issue is out of the scope of this PR.